src/librustc_back/target/arm_linux_androideabi.rs

```diff
@@ -12,7 +12,7 @@ use target::Target;
 
 pub fn target() -> Target {
     let mut base = super::android_base::opts();
-    base.features = "+v7,+vfp3,+d16".to_string();
+    base.features = "+v6".to_string();
```

Comment:
Could this target temporarily stay where it is today for now? We'd like to give a window of time for projects to move over to the armv7 target before we downgrade the arm one. I don't believe we've received a request yet to use arm instead of armv7, so it's not *super* pressing to do so just yet